src/sentry/api/bases/organization.py

```diff
@@ -276,8 +276,13 @@ def get_filter_params(self, request, organization, date_filter_optional=False):
         if not projects:
             raise NoProjects
 
-        environments = [env.name for env in self.get_environments(request, organization)]
-        params = {"start": start, "end": end, "project_id": [p.id for p in projects]}
+        environments = self.get_environments(request, organization)
```

Comment:
This seems like a risky change. Couldn't this be used in many endpoints?

Reply:
oof yeah, this breaks a lot of things. I'll try to find a less risky way to do this :+1: